doc/renderManagerConfig.md

```diff
@@ -73,6 +73,10 @@ This section describes the operation of time warp, which re-projects textures fr
 
 * enabled: True to enable time warp, false to disable it.
 
+* justInTime: If time warp is enabled, true to enable just-in-time warp and false to disable it.  This uses shearing and anisotropic scaling to adjust each scan line so that it appears at the correct location, offsetting the impact of head rotation around the X and Y rotations.  This handles the fact that on some displays scan lines do not all appear at the same time but rather from one edge of the display to another.  This should be turned off for displays that flash all of their pixels at the same time.
+
+* justInTimeRotation: Specifies the edge of the display (as seen by the viewer) where the scanlines begin.  For the OSVR HDK 2.0, this is the bottom of the display (rotation 180).  For the Oculus DK2, this is the right side of the display (rotation 90).  Displays that scan out from the top will have rotation 0 and displays that scan out from the left will have rotation -90.  This does not have to do with how the pixels are sent to the display (which can be set in the operating system) but rather with how the display natively scans out.
+
```

Comment:
The display orientation is indeed a display property.  I was grouping them together to try and keep all of the time-warp settings in one place and I've never had a lot of clarity on which parameters should go in each of these files.  For example, the display rotation and bits-per-color values are stored in a display field within the renderManagerConfig file.

Unfortunately, we can't just use the display rotation directly, because that can be adjusted by the O/S and we really want to know how the physical display scans out rather than how the pixels are oriented.

Glad to move this new rotation to either the display section within renderManagerConfig or to a new section in the display config file.  I don't have a preference so am happy for you to either name the new section in display (or tell me to put it into rendering or one of the other existing sections) and I'll move it.